libraries/AP_ADSB/AP_ADSB.cpp

```diff
@@ -56,7 +56,7 @@ const AP_Param::GroupInfo AP_ADSB::var_info[] = {
     // @Param: TYPE
     // @DisplayName: ADSB Type
     // @Description: Type of ADS-B hardware for ADSB-in and ADSB-out configuration and operation. If any type is selected then MAVLink based ADSB-in messages will always be enabled
-    // @Values: 0:Disabled,1:uAvionix-MAVLink,2:Sagetech,3:uAvionix-UCP,4:Sagetech MX Series
+    // @Values: 0:Disabled,1:uAvionix-MAVLink-InOut,2:Sagetech,3:uAvionix-UCP,4:Sagetech MX Series,5:uAvionix-MAVLink-In
```

Comment:
Feel free to change the order of the Type values here if you want the "-In" option to be higher on the drop-down list.

I guess users won't be confused by which way is "In" (or "out")?

Reply:
Maybe "Rx" and "RxTx"?

Reply:
ADS-B world tends to talk about "ADS-B out" as a sentence, so it seemed slightly clearer to me.